adafruit_as7341.py

```diff
@@ -375,8 +375,18 @@ def channel_680nm(self):
         """The current reading for the 680nm band"""
         self._configure_f5_f8()
         return self._channel_3_data
+      
+    @property
+    def channel_clear(self):
+        """The current reading for the clear sensor"""
+        self._configure_f5_f8()
+        return self._channel_4_data
 
-    # TODO: Add clear and NIR accessors
+    @property
+    def channel_clear(self):
```

Comment:
This one should be named `channel_nir` ?

Reply:
You are right, let me fix that